packages/react-storage/src/components/StorageBrowser/components/Checkbox.tsx

```diff
@@ -20,26 +12,26 @@ interface CheckboxControlProps {
   onSelect?: () => void;
 }
 
-export const CheckboxControl = ({
-  disabled,
+export const Checkbox = ({
```

Comment:
Updated to align with concept of control components being higher level components in charge of transforming contextual data into presentational props.